<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="CadProd_Text">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>Cadastro de Produtos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="background">
      <property name="minimumSize">
       <size>
        <width>459</width>
        <height>700</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(35, 35, 35);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_fake">
         <property name="minimumSize">
          <size>
           <width>419</width>
           <height>653</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);
border-radius: 8px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="paineldeobjetos">
            <property name="minimumSize">
             <size>
              <width>380</width>
              <height>580</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>380</width>
              <height>642</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_ExcNome">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>420</y>
               <width>161</width>
               <height>50</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: rgb(85, 170, 255);
}
QPushButton:hover{
	background-color: rgb(152, 202, 255);
}
QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
             </property>
             <property name="text">
              <string>Excluir por nome</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_Cancelar">
             <property name="geometry">
              <rect>
               <x>110</x>
               <y>540</y>
               <width>161</width>
               <height>50</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: rgb(255, 42, 42);
}
QPushButton:hover{
	background-color: rgb(255, 144, 144);
}
QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
             </property>
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_Cod">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>140</y>
               <width>151</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>CÃ³digo</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_Cod">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>170</y>
               <width>321</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_Nome">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>340</y>
               <width>151</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Nome</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_Nome">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>370</y>
               <width>321</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_ExcCod">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>220</y>
               <width>161</width>
               <height>50</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	background-color: rgb(85, 170, 255);
}
QPushButton:hover{
	background-color: rgb(152, 202, 255);
}
QPushButton:pressed{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}</string>
             </property>
             <property name="text">
              <string>Excluir por cÃ³digo</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_Excluido">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>480</y>
               <width>341</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
